streams/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1,2 @@
+mock-maker-inline
```

Comment:
Why we need this file?

Reply:
I don't think we need this although if we use mocking of statics, we would need to have a dependency on mockito-inline vs mockito-core (like we do in `tools`).

Reply:
Thanks suggestion, I have modified.